lab1.py

```diff
@@ -0,0 +1,34 @@
+import random as rand
+
+def losowa_tablica(n):
+    tab=[]
+    for i in range(n):
+        tab.append(rand.randint(1,100))

```

Comment:
jak juz tworzysz do tego funkcje pomocnicza to wypadaloby zeby przyjmowala zakres losowanych liczb oraz ewentualnie typ losowanych liczb (inty, calkowite, ...). na potrzeby testowania byloby ekstra.